monitoringclient/MonitoringClient.download.recipe

```diff
@@ -32,7 +32,7 @@
             <key>Arguments</key>
             <dict>
                 <key>filename</key>
-                <string>%NAME%-%SUBDOMAIN%.pkg</string>
+                <string>%NAME%-%version%.pkg</string>
```

Comment:
For subscribers who have >1 accounts having the subdomain is very useful. 

We see this in large/multi-regional accounts, and for subscribers who sell [Managed Accounts](https://www.watchmanmonitoring.com/managed-accounts)

Let's keep the subdomain, and put the version after with a hyphen in between.